### DISCUSSION TITLE
Database Sync betwenn 2 Dtabases with diffrent Acces credentials

### DISCUSSION THREAD

**Discussion by pampa-party1** [keepassxreboot/keepassxc#11101] [Q&A] [answered]
Hi,

i have a question about a workaround. I want to build up ab backup/Emergency solution for my database in case i lose my Yubikey. 
I want to setup a clone from my database which is syncing with my actual database. The clone should be at all time the same as the original database. For the clone i want to set up another yubikey which to store in a safe place. Has Anybody an Idea how to set this up?

Best regards

**Comment by droidmonkey** [comment, marked as answer]
Just make a duplicate of your current yubikey. Same secret code on both.

**Reply by pampa-party1** [reply]
Thx for answer. But i have my reasons for this setup. So i´m looking specific for this kind of setup.

**Reply by droidmonkey** [reply]
Your setup is impossible to achieve

**Reply by pampa-party1** [reply]
seems like that. The Solution with a duplicate of your current yubikey is probably then the next best one. but how to do that? from my point of view the secret stays inside the yubikey and is not readable. So how to clone?

**Reply by droidmonkey** [reply]
You'll have to reset your secret on the current one and save it to setup the second one. Unlock database with current yubikey, keep unlocked, reset secret, change yubikey on unlocked database through database settings, save. Make second yubikey with saved secret.
